api/notd/api/models_v1.py

```diff
@@ -288,3 +288,7 @@ class ApiSuperCollectionOverlap(BaseModel):
     otherRegistryAddress: str
     otherRegistryTokenCount: int
     registryTokenCountMap: Dict[str, int]
+
+class ApiMintedTokenCount(BaseModel):
+    date: datetime.date
+    mintedTokenCount: Optional[int]
```

Comment:
this one shouldnt be optional - it should just count so be 0 if there are non filled